monitoring/monitoring_test.go

```diff
@@ -69,10 +69,24 @@ var (
 		"test_gauge",
 		"Testing gauge functionality",
 	)
+
+	testDisabledSum = monitoring.NewSum(
+		"events_disabled_total",
+		"Number of events observed, by name and kind",
+		monitoring.WithLabels(name, kind),
+	)
+
+	testConditionalSum = monitoring.NewSum(
```

Comment:
curious why we register and init seperately. Could we just do `testConditionalSum = monitoring.Register(monitoring.NewSum)`? And RegisterIf where appropriate, of course?

Reply:
I think we can do. Not sure what the history was